README.md

```diff
@@ -1 +1,177 @@
-# desktop-version-portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <h3><b>Portfolio: desktop version</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Portfolio: desktop version <a name="about-project"></a>
+
+**For this project, I established a repository for my portfolio website. My objective is to develop a complete portfolio website that is responsive on both mobile and web platform.**
+
+Desktop version:
+
+![screenshot](/assets/images/desktop.png)
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+The various stacks used for the project includes the following
+<details>
+  <summary>HTMl</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+<details>
+<summary>Linters</summary>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Learn how to translate a Figma design into a functional UI.**
+- **Utilize Flexbox to position various elements on a web page.**
+- **Utilize CSS Grid to arrange different elements on a web page.**
+- **Develop a portfolio website.**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+on MAC:
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Install
+
+Install this project with:
+
+
+Example command:
+
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+### Run tests
+
+To run tests, run the following command:
```

Comment:
- [ ] Have you done tests for this app? If not please remove this section. Please 🙏🏽 complete this section describing the steps that a common user should follow to use your app.